### PR TITLE
Updating to latest USWDS

### PR DESCRIPTION
**Pull request by line47**
👀  Preview - https://federalist-proxy.app.cloud.gov/preview/18f/federalist-uswds-template/sw-update-latest/

See USWDS release notes here - https://github.com/18F/uswds-jekyll/releases

### CODE REVIEW

File: assets/css/main.scss [hunk 1]
```diff
@@ -5,8 +5,3 @@ $font-path: '../uswds/fonts';
 $image-path: '../uswds/img';
 
 @import 'uswds/all';
-
-// fix for overlay flashing on page reload
```
Comment (line47) [inline assets/css/main.scss:9]: fixed upstream in the US Web Design repo

File: _includes/components/header--extended.html [hunk 2]
```diff
@@ -1,85 +0,0 @@
-{% if header %}
```
Comment (line47) [inline _includes/components/header--extended.html:1]: This file is no longer needed as the uswds-jekyll version of this file has been updated

**Review by jseppi — APPROVED** [review]
LGTM

**Review by wslack — APPROVED** [review]
No objections, but hard to know what the major changes are for this template (relative to the general USWDS changes)

**Comment by line47** [comment]
It's really just inheriting any changes made in uswds-jekyll in this case it was the updates listed here - https://github.com/18F/uswds-jekyll/releases/tag/v2.2.0

**Pull request merged**